shell/src/main/java/alluxio/cli/fsadmin/command/ReportCommand.java

```diff
@@ -44,8 +47,37 @@
   private BlockMasterClient mBlockMasterClient;
   private PrintStream mPrintStream;
 
+  private static final Option LIVE_OPTION =
+      Option.builder("live")
+          .required(false)
+          .hasArg(false)
+          .desc("show capacity information of live workers.")
+          .build();
+
+  private static final Option LOST_OPTION =
+      Option.builder("lost")
+          .required(false)
+          .hasArg(false)
+          .desc("show capacity information of lost workers.")
+          .build();
+
+  private static final Option SPECIFIED_OPTION =
+      Option.builder("worker")
```

Comment:
should this be called "workers"? We might also want to store the option names in string constants since they are used in multiple places and need to match

Reply:
Renamed to "workers", thanks
ReportCommand
```java
  public static final String LIVE_OPTION_NAME = "live";
  public static final String LOST_OPTION_NAME = "lost";
  public static final String SPECIFIED_OPTION_NAME = "workers";
  public static final String HELP_OPTION_NAME = "h";
```

So in CapacityCommand, I can also get the option name to check
```java
 if (cl.hasOption(ReportCommand.LIVE_OPTION_NAME)) {
```